id.go

```diff
@@ -0,0 +1,5 @@
+package imap
+
+type IdData struct {
+	Data map[string]string
```

Comment:
Can we use struct fields for the stuff defined in the RFC?

Reply:
@emersion No

Reply:
You're going to have to give me more details if you want me to understand your reply. In other words, please stick to constructive comments.